e2e_test/batch/functions/array_min.slt.part

```diff
@@ -0,0 +1,47 @@
+query I
+select array_min(array[1, 2, 3]);
+----
+1
+
+query I
+select array_min(array[2, 3, 5, 2, 4]);
+----
+2
+
+query I
+select array_min(array[114514, 123456]);
+----
+114514
+
+query I
+select array_min(array['a', 'b', 'c', 'a']);
+----
+a
+
+query I
+select array_min(array['💩', '🤔️', '🥵', '🥳', '💩']);
+----
+💩
```

Comment:
Not sure if this is a proper case because we can't tell which unicode is the smallest. 😅

Reply:
Tried in Rust playground and found this🤣
```rust
'🤔' -> U+1F914
'🥵' -> U+1F975
'💩' -> U+1F4A9
```
The default comparison should be led by lexicographic order I think?

Reply:
Yes. I mean, reviewers don't know that so they can't tell this case is correct at a glance.